ARM_DisassemblerExplorer/src/parser/ARM/ARM_Instruction.h

```diff
@@ -0,0 +1,45 @@
+/**
+ * @addtogroup  Parser
+ * @{
+ * @file        ARM_Instruction.h
+ * @author      Pascal-Emmanuel Lachance
+ * @p           https://www.github.com/Raesangur
+ * @date        2020/07/24  -  19:51
+ *
+ * @brief       Base pure virtual class onto which all ARM assembly instructions are based.
+ */
```

Comment:
I made a header that includes the GPLv3 in it, I would like it being used instead for that project, if you don't mind

Reply:
sure

Reply:
Do you need it?

Reply:
ill copy it from one of your header files